packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -373,3 +374,49 @@ test.describe('`isDismissible` prop', () => {
         });
     });
 });
+
+test.describe('isOpen prop', () => {
+    test('should not render when isOpen = false', async ({ mount, page }) => {
```

Comment:
just a nit here, the component _will_ render. It just won't open.

Reply:
@jamieomaguire  does `'should not render open when isOpen = false'` sound ok? Not sure how best to phrase it.

Reply:
Or "should not be visible ..." because that's the playwright method we're using